arangod/IResearch/IResearchKludge.cpp

```diff
@@ -24,16 +24,11 @@
 
 #include "IResearchKludge.h"
 
-#include "utils/thread_utils.hpp"
-
-#include "Basics/Common.h"
-
-using namespace std::chrono_literals;
+#include <string>
+#include <string_view>
 
 namespace {
 
-constexpr auto RW_MUTEX_WAIT_TIMEOUT = 50ms;
```

Comment:
The key goal is to remove this synchronization primitive, which works only by time for us.